semgrep/tests/e2e/snapshots/test_rule_parser/test_rule_parser__failure__error_messages/badlanguage/error-in-color.txt

```diff
@@ -5,5 +5,5 @@
 [94m  | [0m               [31m^^^^^^^^^^[0m
 [94m8 | [0m    severity: WARNING
 
-[31munsupported language: intercal. supported languages are: C#, C++, HCL, JSON, Json, Kotlin, Rust, Yaml, bash, c, cpp, cs, csharp, generic, go, golang, hack, hacklang, hcl, html, java, javascript, js, json, kotlin, kt, lua, ml, none, ocaml, php, py, python, python2, python3, rb, regex, rs, ruby, rust, scala, sh, terraform, tf, ts, typescript, vue, yaml[0m
+[31munsupported language: intercal. supported languages are: bash, c, c#, c++, cpp, csharp, generic, go, golang, hack, hcl, html, java, javascript, js, json, kotlin, kt, lua, none, ocaml, php, py, python, python2, python3, r, regex, ruby, rust, scala, sh, terraform, tf, ts, typescript, vue, yaml[0m
```

Comment:
Hmm I'm not sure `ml`, `rb`, or `rs` make sense here. And I don't think `hacklang` is valid in semgrep-core.

Reply:
Also this is now case-insensitive.

Reply:
I'll investigate to see if anyone is already using the four short abbreviations. Confirmed that `hacklang` is invalid in `Lang.ml`.

Reply:
This is the current exhaustive list used in semgrep-rules:

`['C#', 'bash', 'c', 'csharp', 'generic', 'go', 'hcl', 'java', 'javascript', 'js', 'json', 'kotlin', 'kt', 'lua', 'ocaml', 'php', 'python', 'regex', 'ruby', 'rust', 'scala', 'sh', 'ts', 'typescript', 'yaml']`